package.json

```diff
@@ -208,13 +213,13 @@
 	"scripts": {
 		"vscode:prepublish": "webpack --mode production",
 		"compile": "webpack --mode none",
-		"watch": "webpack --mode development --watch --info-verbosity verbose",
+		"watch": "tsc src/extension.ts --watch",
```

Comment:
Webpack compiles the typescript code for us, so we should leave this as-is. More on webpack here: https://code.visualstudio.com/api/working-with-extensions/bundling-extension#using-webpack